From cfbeab2a4421ca0ba8f19d3958abe9e2fc5898bc Mon Sep 17 00:00:00 2001
From: Jakub Zawadzki <darkjames-ws@darkjames.pl>
Date: Tue, 28 Aug 2012 08:44:41 +0000
Subject: [PATCH 2/4] Fix bug #7668

Use correct field type for lcp.opt.oui

svn path=/trunk/; revision=44688
---
 epan/dissectors/packet-ppp.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/epan/dissectors/packet-ppp.c b/epan/dissectors/packet-ppp.c
index ca740c5..6109313 100644
--- a/epan/dissectors/packet-ppp.c
+++ b/epan/dissectors/packet-ppp.c
@@ -5570,7 +5570,7 @@ proto_register_lcp(void)
             { "Length", "lcp.opt.length", FT_UINT8, BASE_DEC,
                 NULL, 0x0, NULL, HFILL }},
         { &hf_lcp_opt_oui,
-            { "OUI", "lcp.opt.oui", FT_BYTES, BASE_NONE,
+            { "OUI", "lcp.opt.oui", FT_UINT24, BASE_HEX,
                 NULL, 0x0, NULL, HFILL }},
         { &hf_lcp_opt_kind,
             { "Kind", "lcp.opt.kind", FT_UINT8, BASE_DEC_HEX,
-- 
1.7.10.4

